### ISSUE TITLE
[feature] Use schema store to get json schema

### ISSUE THREAD

**Issue by Freed-Wu** [kkiyama117/coc-toml#115]
https://github.com/kkiyama117/coc-toml/blob/main/src/index.ts#L64

Why not use [schemastore](https://www.schemastore.org/) to get json schema?

Related issue: <https://github.com/neoclide/coc-json/issues/75>

**Comment by Freed-Wu** [comment]
<https://taplo.tamasfe.dev/configuration/developing-schemas.html#publishing>:

> Submitting schemas directly to Taplo is not possible anymore, and all JSON schemas should be submitted to the JSON Schema Store.

**Comment by Freed-Wu** [comment]
Yes, <https://github.com/tamasfe/taplo/tree/master/editors/vscode> use schemastore.

**Comment by kkiyama117** [comment]
I think it is a very good suggestion, as are the other issues you have put up. I will work on it later if I can find the time, but I am currently a university student and I don't think I will have time to work on it for another month or so due to my graduation thesis, remaining credits and graduate exams.
This `coc-toml` is based on the alpha version of `taplo`'s plugin for vscode, modified to fit the configuration of other coc plugins. If I remember correctly, taplo is now in a different major version from that time, and I think there are discrepancies in some functions and links such as schema, etc. because the internal structure is different. If you want these features immediately and can use Node.js, I hope you can make the changes and create a PR.

**Comment by kkiyama117** [comment]
Also, I now use a plugin structure in vim based on the [`denops`](https://github.com/vim-denops/denops.vim) ecosystem and do not use `coc` in main. That, and the private reasons mentioned above, have slowed down the updating of coc-toml. If you use coc regularly and you or someone you know is familiar with coc, nodejs or vim, and you are interested, please let me know in a separate issue as I will transfer this repository.

**Comment by Freed-Wu** [comment]
> someone you know is familiar with coc, nodejs or vim

How about <https://github.com/coc-extensions>?

**Comment by kkiyama117** [comment]
Sure. I'll discuss it there when I can take the time to reply properly.

**Comment by kkiyama117** [comment]
Now we use `Tombi` as TOML LSP engine, and Tombi supports schema store as default. we can use store now.

**Comment by Freed-Wu** [comment]
> > someone you know is familiar with coc, nodejs or vim
> 
> How about <https://github.com/coc-extensions>?

https://github.com/neoclide/coc.nvim/discussions/5593
mentioned <https://github.com/coc-plugin>, should be a new choice.

**Comment by kkiyama117** [comment]
@Freed-Wu now I start to delegate the ownership of this repo, so I think the maintenance of it becomes more active. Thanks.

**Comment by kkiyama117** [comment]
https://github.com/neoclide/coc.nvim/issues/5752
